.travis.yml

```diff
@@ -5,13 +5,15 @@ matrix:
     - python: 3.4
     - python: 3.5
     - python: 3.6
+    - python: 3.6
+      env: TOXENV=flake8
     # see the following issue on why 3.7 is weirdo
     # https://github.com/travis-ci/travis-ci/issues/9815
     - python: 3.7
       dist: xenial
       sudo: true
 install:
-- pip install -r requirements-travis.txt
+- pip install --no-cache-dir -r requirements-travis.txt
```

Comment:
This prevents pip from attempting to store these. It makes the tox install portion slightly slower but I'm looking at how to remove the translation complication from the travis script portion and move it either into the tox command or the tests that need the translations.